test/Altinn.App.Api.Tests/CustomWebApplicationFactory.cs

```diff
@@ -116,6 +125,38 @@ private void ConfigureFakeHttpClientHandler(IServiceCollection services)
         ));
     }
 
+    /// <summary>
+    /// Set this in your test class constructor
+    /// </summary>
+    protected async Task<T> VerifyStatusAndDeserialize<T>(
+        HttpResponseMessage response,
+        HttpStatusCode expectedStatusCode
+    )
+    {
+        // Verify status code
+        response.Should().HaveStatusCode(expectedStatusCode);
+
+        // Deserialize content and log everything if it fails
+        var content = await response.Content.ReadAsStringAsync();
+        try
+        {
+            return JsonSerializer.Deserialize<T>(content, _jsonSerializerOptions)
+                ?? throw new JsonException("Content was \"null\"");
+        }
+        catch (Exception)
+        {
+            _outputHelper.WriteLine(string.Empty);
```

Comment:
Hvorfor skrive ut string.Empty?

Reply:
I hope it was because `.WriteLine()` wasn't defined and `.WriteLine("")` caused a warning when I wanted more space between lines to make the failed request response stand out more in the logs.